### DISCUSSION TITLE
Default account secret after the first installation of docker

### DISCUSSION THREAD

**Discussion by Jengro777** [uxbox/uxbox#2999] [Q&A]
After the first installation of Docker, whether there is a default account password or not, the mailbox cannot be registered.

**Comment by niwinz** [comment]
There are no default account and no default password. On default installation email verification should be disabled (if you used the current docker compose configuration) so you should be able to register a new account without having SMTP enabled. In any case, the docker compose already comes with example mail catcher service that allows setup a dummy smtp receiver for read all emails that penpot platform send.
